sw/device/tests/crypto/ecdh_p256_functest.c

```diff
@@ -3,52 +3,148 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include "sw/device/lib/crypto/drivers/otbn.h"
-#include "sw/device/lib/crypto/impl/ecc/ecdh_p256.h"
-#include "sw/device/lib/crypto/impl/status.h"
+#include "sw/device/lib/crypto/impl/integrity.h"
+#include "sw/device/lib/crypto/impl/keyblob.h"
+#include "sw/device/lib/crypto/include/ecc.h"
 #include "sw/device/lib/runtime/log.h"
 #include "sw/device/lib/testing/entropy_testutils.h"
 #include "sw/device/lib/testing/test_framework/check.h"
 #include "sw/device/lib/testing/test_framework/ottf_main.h"
 
+static const ecc_curve_t kCurveP256 = {
+    .curve_type = kEccCurveTypeNistP256,
+    .domain_parameter = NULL,
+};
+
+// Configuration for the private key.
+static const crypto_key_config_t kEcdhPrivateKeyConfig = {
+    .version = kCryptoLibVersion1,
+    .key_mode = kKeyModeEcdh,
+    .key_length = 256 / 8,
```

Comment:
nit: `256 / 8` and `256 / 32` can be mapped to anonymous enums to improve readability.  Ok to leave as is if you think this is good enough.

Reply:
Changed to import the `p256_common.h` internal file in the test, which has these constants declared in enums.